README.md

```diff
@@ -54,7 +54,7 @@ What does the seed give you out of the box?
 |demo/tests| The tests for your plugin|
 |src| The plugin source code|
 |src/platform/android| Plugin Android specific configuration|
-|src/platform/ios|Plugin ios specific configuration|
+|src/platform/ios|Plugin ios specific configuration. All configurations that are applicable via XCode Info tab and are saved in application's Info.plist file can also be applied for plugins. |
```

Comment:
In the article mentioned there is a link to [this page](https://developer.apple.com/library/content/documentation/General/Reference/InfoPlistKeyReference/Articles/AboutInformationPropertyListFiles.html) . It says:

"For more information about the format of Info.plist, see About Information Property List Files."

Do you think in this case more clarification is needed?